foundation-extensions/mobile-core/identity/identity-api-reference.md

```diff
@@ -2503,27 +2565,24 @@ ACPIdentity.SyncIdentifiers(ids);
 {% endtab %}
 {% endtabs %}
 
-## syncIdentifiers \(overloaded\)
+## syncIdentifiers (overloaded)
 
 The function of this API is the same as the `syncIdentifier` API. This API passes a list of identifiers, and each identifier contains an `identifier type` as the key and an `identifier` as the value. In each identifier pair, if the `identifier type` contains a null or an empty string, the identifier is ignored by the Identity extension.
 
-Starting with _ACPIdentity v2.1.3 \(iOS\)_ and _Identity v1.1.2 \(Android\)_ if the new `identifier` value is null or empty, this ID type is removed from the local storage, Identity shared state and not synced with the Adobe ECID Service.
+Starting with _ACPIdentity v2.1.3 (iOS)_ and _Identity v1.1.2 (Android)_ if the new `identifier` value is null or empty, this ID type is removed from the local storage, Identity shared state and not synced with the Adobe Experience Cloud ID (ECID) Service.
```

Comment:
let's simplify `Adobe Experience Cloud ID (ECID) Service` to Experience Cloud ID Service or simple ECID Service.

Reply:
I've been using "Adobe Experience Cloud Identity Service". It looks like I missed changing this instance. I'll update it to "Adobe Experience Cloud Identity Service".